dune-project

```diff
@@ -368,6 +368,11 @@ functions.
     (process_limits (>= "1.5.5"))
   )
 )
+(package (name source)
+  (synopsis "Types for tracking the source of scanned content")
```

Comment:
I would move this code in lib_parsing/ for now, close to Loc.ml, Tok.ml, etc.
No need for one library with just one module.

Reply:
Done in 3f206d0db